src/test/groovy/net/fabricmc/loom/MixinVersionTest.groovy

```diff
@@ -0,0 +1,21 @@
+package net.fabricmc.loom
+
+import spock.lang.Specification
+
+class MixinVersionTest extends Specification {
+
+	def "Mixin version parsing"() {
+		expect:
+			LoomGradleExtension.parseMixinVersion(input) == output
+
+		where:
+			input					| output
+			"0.7.11.10"				| "0.7"
+			"0.8.2+build.24"		| "0.8"
```

Comment:
Shouldn't these return 0.8.2, I do see versions with min versions of 0.8.2

Reply:
Thinking about it, it might just be best to remove this, the example mod sets it and im not even sure its actaully checked.

Reply:
It is actually checked, but not that useful in the context of fabric as the loader determines the mixin version, mods should just declare the minimum loader version in their mod json.